policyengine_us/parameters/gov/states/wv/tax/income/subtractions/senior_citizen_disability_deduction/total_modification.yaml

```diff
@@ -0,0 +1,22 @@
+description: West Virginia uses these sources as total modification as a limit for senior citizen or disability deduction.
+
+values:
+  1986-01-01:
+    - us_govt_interest # Line 27
+    - CERTAIN FEDERAL LAW ENFORCEMENT RETIREMENT # Line 28
+    - WEST VIRGINIA STATE OR LOCAL POLICE, DEPUTY SHERIFFS’ OR FIREMEN’S RETIREMENT # Line 29
```

Comment:
```suggestion
    - WEST VIRGINIA STATE OR LOCAL POLICE, DEPUTY SHERIFFS’ OR FIREMEN’S RETIREMENT # Line 29
``` for the variables that are not in the system, write them as comment 
# WEST VIRGINIA STATE OR LOCAL POLICE, DEPUTY SHERIFFS’ OR FIREMEN’S RETIREMENT